docs/contributing.rst

```diff
@@ -184,8 +184,10 @@ To release a new version of the theme, core team will take the following steps:
 #. Update the changelog (``docs/changelog.rst``) with the version information.
 #. Run ``python setup.py update_translations`` to compile new translation files
    and update Transifex.
-#. Run ``python setup.py build_assets`` to rebuild all the theme assets and the Python
-   package.
+#. Run ``npm run build`` to rebuild all the theme assets and the Python
+   package, notice that ``package-lock.json`` will be updated with a new package
+   version. But take care that it isn't also bumping versions of dependencies
+   in ways that are risky to the release.
```

Comment:
I don't follow this. What are the concerns here?

Reply:
Since `package-lock.json` shouldn't be updated by bump2version, I wanted to include an extra reflection that it's actually generated while running the other `npm` commands and that someone doing a release should pay attention to that.

Reply:
This might not be too actionable I suppose, unless we're telling the release manager what to look for specifically. I think this falls under testing more than anything :shrug: